### PR TITLE
Document bulk user upload

### PR DESCRIPTION
**Pull request by m5r**
Related code PR: medic/cht-core#7678

### CODE REVIEW

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
Yowza - this looks awesome!  thanks for this.

I'll need to dive in deeper, but right away I see two changes we need:
* please use english screenshots, not french. I think you can easily grab the english ones [from here](https://docs.google.com/document/d/1HwyZUl6w8YEEK6W9r4qMn_0nzrujHKeHNq5-49dzICg/edit#heading=h.bt4f1ry64jmk)
* We need to specify this will only work in 3.16 or later

more to come!

**Comment by m5r** [comment]
> - please use english screenshots, not french. I think you can easily grab the english ones [from here](https://docs.google.com/document/d/1HwyZUl6w8YEEK6W9r4qMn_0nzrujHKeHNq5-49dzICg/edit#heading=h.bt4f1ry64jmk)
> - We need to specify this will only work in 3.16 or later

Done and done, thanks @mrjones-plip  
You really did a great job documenting the steps to use this feature!

**Comment by mrjones-plip** [comment]
Awesome! I'll dive back into this today or tomorrow.

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
Yay - getting closer!

I did another pass on the content and made some suggestions.  Biggest missing piece of content I think is the API area for the bare minimum set of fields to create different objects.  See comments. 

Also, looking over the images, I see we need to redo them as they're incongruous and  reference some specific deployments by name and specific, real geographic places.  I'll do this tomorrow!

**Comment by mrjones-plip** [comment]
I uploaded new screenshots all with a 1px border, all on the same dataset.

**Comment by mrjones-plip** [comment]
@m5r  - we should try and push this through before 3.16 goes live! Lemme know if you wanna do a real-time session to save on some back and forth time on the PR

File: content/en/apps/reference/api.md [hunk 1]
```diff
@@ -1040,6 +1040,147 @@ Content-Type: application/json
...
+Creating users from a CSV file behaves the same as passing a JSON array of users into the [`POST /api/v1/users`]({{< ref "apps/reference/api#post-apiv1users" >}})
+where a row represents a user object and a column represents a user object property.  
+Columns with a `:excluded` suffix will be ignored, this allows providing a more user-friendly experience with
+autocompletion on fields or dealing with names instead of long, unreadable ids.
+
+In order to facilitate this process, we have made available a spreadsheet compatible with the `default` configuration of the CHT.
+[Click here](https://docs.google.com/spreadsheets/d/1yUenFP-5deQ0I9c-OYDTpbKYrkl3juv9djXoLLPoQ7Y/copy) to make a copy of the spreadsheet in Google Sheets.  
+[A guide]({{< ref "apps/guides/data/users-bulk-load" >}}) on how to import users with this spreadsheet from within the Admin Console is available
+in case you don't want to interact with this API yourself.
+
```
[the remark this answers is not in the document]
Comment (m5r) [reply content/en/apps/reference/api.md:1057]: That would be the [Supported Properties](https://docs.communityhealthtoolkit.org/apps/reference/api/#supported-properties-2) section in the Users section. I agree they could use some love, I had to rely on the implementation and some examples to find what fields I can use in the `contact` and `place` properties.

Do we have an example of documentation where we list out all the possible fields with nested properties?

**Review by mrjones-plip — APPROVED** [review]
While I do think we should document all the possible fields, I think what we have is usable and accurate.  Let's ship this and iteratively improve on it.

I think a good next iteration would be a example spreadsheets that show how to just add users and just add users and contacts.  In conjunction with the one we have, this will give users a great working example on how each works.

We can revisit the exhaustive list fields!

excellent work on seeing this through - thank you!

**Pull request merged**